lib/api.js

```diff
@@ -47,7 +48,7 @@ exports.default = function (_ref) {
   if (!module.exports) {
     // babel did not transform modules
     // eslint-disable-next-line no-underscore-dangle
-    component = module.__esModule ? module.default : module;
+    component = ctx.__esModule ? ctx.default : ctx;
```

Comment:
That's a good question. I guess we should just take care of the default export (I think that's how vue-loader works, too).